package.json

```diff
@@ -4,7 +4,6 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "pretest": "gulp lint",
     "test": "nyc ava"
```

Comment:
When fixing stuff, I often run the tests and find it annoying, to be always compliant to the eslint rules. E.g. I quickly comment something out to test that, but then eslint grumbles about unused variables and whatnot.
I'd prefer to set _gulp lint_ as a precommit hook.